services/ansibler/server/server.go

```diff
@@ -156,7 +161,7 @@ func (*server) SetUpLoadbalancers(_ context.Context, req *pb.SetUpLBRequest) (*p
 	for _, lb := range req.DesiredState.LoadBalancerClusters {
 		np, ok := k8sNodepools[lb.TargetedK8S]
 		if !ok {
-			log.Error().Msgf("Loadbalancer %s has not found a target k8s cluster (%s)", lb.ClusterInfo.Name, lb.TargetedK8S)
+			log.Info().Msgf("Loadbalancer %s has not found a target k8s cluster (%s)", lb.ClusterInfo.Name, lb.TargetedK8S)
```

Comment:
Isn't this considered an error/warning? If we have a Loadbalancer in a state which does not target any Kubernetes cluster, it should be caught by the frontend, thus if we detect it here, something went wrong.